README.md

```diff
@@ -1,3 +1,28 @@
+# 1. Introduction
+
+This web application is a [Google Summer of Code 2019](https://summerofcode.withgoogle.com) project under the [MacPorts](https://www.macports.org) organisation.
+
+**Student**: [Arjun Salyan](https://github.com/arjunsalyan) <br>
+**Mentors**: [Mojca Miklavec](https://github.com/mojca), [Umesh Singla](https://github.com/umeshksingla) <br>
+
+### Architecture of the Project
```

Comment:
This could be the second paragraph. The last one could be about who wrote the app.

Reply:
Shall I drop the headings?